examples/wordcount/flink-operator-custom-resource.yaml

```diff
@@ -8,23 +8,27 @@ metadata:
     environment: development
 spec:
   image: docker.io/lyft/wordcount-operator-example:{sha}
+  deleteMode: None
```

Comment:
Why was this added?

Reply:
The job finishes and the operator tries to take a savepoint. That's a bug that needs to be fixed separately.

For the demo, the application should just be deleted.

Reply:
Ah, it has a bounded input. We should probably fix that, as the operator doesn't (intentionally) support jobs that complete.

Reply:
I actually don't see why the operator shouldn't support such jobs. Created https://github.com/lyft/flinkk8soperator/issues/138 to follow up on this.

Reply:
Yeah, I mean any support currently in the operator is accidental; we haven't tested this use case or thought deeply about how to support it. Would be good to have better support but it's not a use case internally.